packages/math-input/src/components/keypad/mobile-keypad.tsx

```diff
@@ -178,31 +178,28 @@ class MobileKeypad extends React.Component<Props, State> implements KeypadAPI {
         const convertDotToTimes = keypadConfig?.times;
 
         return (
-            <AphroditeCssTransitionGroup
-                transitionEnterTimeout={AnimationDurationInMS}
-                transitionLeaveTimeout={AnimationDurationInMS}
-                transitionStyle={{
-                    enter: {
-                        transform: "translate3d(0, 100%, 0)",
-                        transition: `${AnimationDurationInMS}ms ease-out`,
-                    },
-                    enterActive: {
-                        transform: "translate3d(0, 0, 0)",
-                    },
-                    leave: {
-                        transform: "translate3d(0, 0, 0)",
-                        transition: `${AnimationDurationInMS}ms ease-out`,
-                    },
-                    leaveActive: {
-                        transform: "translate3d(0, 100%, 0)",
-                    },
-                }}
-            >
-                {keypadActive ? (
-                    <View
-                        style={containerStyle}
-                        forwardRef={this._containerRef}
-                    >
+            <View style={containerStyle} forwardRef={this._containerRef}>
```

Comment:
Just want to double-check that this was tested in webapp.  I feel like I tried this and ran into some layout issues.  I'm guessing this means that we'll always have a `<div />` that is the "visible size/position" of the keypad and the keypad will animate up/down within that div.

Reply:
Still testing as I needed the PR to test :) 

Sadly after testing -- this only fixes the null reference issue I was having in scrollElementIntoViewWithKeypad. It doesn't break anything else, but it still doesn't allow us to scroll our inputs into view. The `keypadElement.getBoundingRect().height` seems to be returning zero as there's no content available to give the necessary height for our calculations at the time of scroll.

Reply:
This was solved in the scrollableElement PR! We're going to use a setTimeout in order to ensure we get the height AFTER the keypad has been enabled.